src/python/grpcio/grpc/_simple_stubs.py

```diff
@@ -0,0 +1,440 @@
+# Copyright 2020 The gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Functions that obviate explicit stubs and explicit channels."""
+
+import collections
+import datetime
+import os
+import logging
+import threading
+from typing import Any, AnyStr, Callable, Iterator, Optional, Sequence, Tuple, TypeVar, Union
+
+import grpc
+
+_LOGGER = logging.getLogger(__name__)
```

Comment:
Could we reduce the amount of log for this `_LOGGER`? E.g., use `_LOGGER.debug`, or set default level to `logging.WARN`?

Reply:
You make a good point. I think we've somewhat been neglecting add logs to the project. Currently, we really only log when there's an error. It wouldn't be useful for us to tell a user to turn up their Python logging level because it would give us no information. I'll bump these to `logging.DEBUG`, but let's also make an attempt to add more such logs in the future now that we've thought about it a bit.